engine/schema/src/main/java/com/cloud/offerings/NetworkOfferingVO.java

```diff
@@ -165,6 +165,17 @@ public String getDisplayText() {
     @Column(name="service_package_id")
     String servicePackageUuid = null;
 
+    @Column(name = "select_snat_address_allowed")
```

Comment:
Can we use network offering details instead of a new column ?
well, it means lots of changes maybe

Reply:
this would be extra work and this functionality is rather generic. It is not so that we want to allow this with one network plugin and not with the other.
So unless there is a good argument against it, I would like to keep it as is.

Reply:
ok to me. @DaanHoogland 
can you rename the column name as well ?